test/sol6/reserves/kyberFprReserveV2.js

```diff
@@ -0,0 +1,2902 @@
+const ConversionRates = artifacts.require("MockEnhancedStepFunctions.sol");
+const TestToken = artifacts.require("TestToken.sol");
+const WethToken = artifacts.require("WethToken.sol");
+const Reserve = artifacts.require("KyberFprReserveV2");
+const MockSanityRates = artifacts.require("MockSanityRates");
+const NoPayableFallback = artifacts.require("NoPayableFallback");
+
```

Comment:
wonder if this reserve should be able to work with apr conversion rate.

I think not, but lets make sure.

Reply:
yes
not relevant.